sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -287,7 +287,7 @@ Azure::Response<std::vector<uint8_t>> KeyClient::BackupKey(
 
   // Convert the internal KeyBackup model to a raw vector<uint8_t>.
   return Azure::Response<std::vector<uint8_t>>(
-      response.ExtractValue().Value, response.ExtractRawResponse());
+      response.Value.Value, std::move(response.RawResponse));
```

Comment:
@vhvb1989 we may want to rename the `Value` property on `KeyBackup`, since `.Value.Value` reads as quite confusing. It's in `_detail` btw, so not critical:
https://github.com/Azure/azure-sdk-for-cpp/blob/ddd7a093f44cc997d1ffc66fd9a018b43c965e45/sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/details/key_backup.hpp#L20-L23